### PR TITLE
Markdown conversion for `ru` - Replace - XPath section :warning: Do not squash merge :warning: 

### PR DESCRIPTION
**Pull request by SphinxKnight**
No conversion report as there were no error

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Please :warning: Do not squash merge :warning:

File: files/ru/web/xpath/functions/floor/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+### Определено
+
+[XPath 1.0 4.4](http://www.w3.org/TR/xpath#function-floor)
+
+### Поддержка Gecko
+
+Поддерживается
+
+{{ languages( { "fr": "fr/XPath/Fonctions/floor", "ja": "ja/XPath/Functions/floor", "pl": "pl/XPath/Funkcje/floor", "en": "en/XPath/Fonctions/floor" } ) }}
```
Comment (sashasushko) [inline files/ru/web/xpath/functions/floor/index.md:34]: Do we need this macro?

Comment (SphinxKnight) [reply files/ru/web/xpath/functions/floor/index.md:34]: No we don't, `{{languages}}` can be removed (it's a source of possible later errors due to https://github.com/mdn/yari/issues/4744 and https://github.com/mdn/translated-content/issues/5603

Comment (SphinxKnight) [reply files/ru/web/xpath/functions/floor/index.md:34]: I'd say such a removal may has its place in a "cleanup" PR. Do you want me to open a PR to remove `{{languages}}` for this section and rebase this current PR on top of it?

**Review by sashasushko — APPROVED** [review]

**Pull request merged**